api_practice/D:\py\pycharmProjects\api\log_msg.log

```diff
@@ -0,0 +1,1098 @@
+INFO     : 2022-07-17 08:43:35,619 : [server.py : 75] : Started server process [21262]
```

Comment:
This log file must not be checked-in at all. Please delete it.

Reply:
file deleted